GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2593,7 +2593,7 @@ private void LaunchRebase(string command)
 
             string rebaseCmd = GitCommandHelpers.RebaseCmd(
                 LatestSelectedRevision.FirstParentId?.ToString(),
-                interactive: true, preserveMerges: false, autosquash: false, autoStash: true);
+                interactive: true, preserveMerges: false, autosquash: false, autoStash: true, ignoreDate: true);
```

Comment:
This is too intrusive IMO because `interactive` and `ignoreDate` conflict with each other.

Reply:
I.e. it overrides the previous behavior because `ignoreDate` takes precedence.
```suggestion
                interactive: true, preserveMerges: false, autosquash: false, autoStash: true, ignoreDate: false);
```